js/client/modules/@arangodb/testsuites/endpoints.js

```diff
@@ -279,10 +288,14 @@ class endpointRunner extends tu.runInArangoshRunner {
       }
       sleep(2);
       try {
-        obj.instance.checkArangoConnection(20);
+        obj.instance.checkArangoConnection(15, true);
       } catch (ex) {
+        fatalError = true;
         print(RED + Date() + ' Server did not become available on time' + RESET);
         obj.instance.shutdownArangod(true);
+        crashUtils.aggregateDebugger(obj.instance, obj.options);
+        obj.instance.waitForExitAfterDebugKill();
+
```

Comment:
@dothebart 
Looks like `fatalError` is set but can not be read after this code? Because we have `return` on line 305

Reply:
its this loop: `return Object.keys(endpoints).reduce((results, endpointName) => {`

Reply:
So here we return only from current iteration?

Reply:
yes, exactly.